x-pack/test/functional/services/ml/trained_models_table.ts

```diff
@@ -218,6 +222,7 @@ export function TrainedModelsTableProvider({ getService }: FtrProviderContext) {
           await testSubjects.existOrFail('mlTrainedModelRowDetails', { timeout: 1000 });
         }
       });
+      await mlCommonUI.waitForRefreshButtonEnabled();
```

Comment:
Not sure what we get from this wait at the end of the method. Is it there to make sure that follow-up steps execute properly?

Reply:
It's to make sure that all ongoing requests are completed after expanding a row. At the moment it calls the `_stats` API.